pdfannots/printer/json.py

```diff
@@ -15,30 +15,19 @@ def annot_to_dict(
 
     result = {
         "type": annot.subtype.name,
-        "page": annot.pos.page.pageno + 1,
+        "page_number": annot.pos.page.pageno + 1,
```

Comment:
Actually, there is one thing I'm uneasy with here. I'd prefer to keep "page" as is, to avoid breaking people that may already depend on the JSON format output from this script.